modules/nf-core/hifiasm/main.nf

```diff
@@ -2,16 +2,19 @@ process HIFIASM {
     tag "$meta.id"
     label 'process_high'
 
-    conda "bioconda::hifiasm=0.15.4"
+    conda "bioconda::hifiasm=0.18.5"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/hifiasm:0.15.4--h2e03b76_0' :
-        'quay.io/biocontainers/hifiasm:0.15.4--h2e03b76_0' }"
+        'https://depot.galaxyproject.org/singularity/hifiasm:0.18.5--h5b5514e_0' :
+        'quay.io/biocontainers/hifiasm:0.18.5--h5b5514e_0' }"
 
     input:
     tuple val(meta), path(reads)
     path  paternal_kmer_dump
     path  maternal_kmer_dump
     val   use_parental_kmers
+    path  hic_read1
+    path  hic_read2
+    val   use_hic
```

Comment:
I know this is the same style as the parental-kmer mode above, but the nf-core guidelines advise that only files are passed through the input channels.
Could you remove `use_hic` here and infer it in the script section below instead ? At the same time you can check that you're not in a situation where only one of `hic_read1` and `hic_read2` is defined.